data/json/furniture.json

```diff
@@ -1182,7 +1182,7 @@
     "id": "f_locker",
     "name": "locker",
     "symbol": "{",
-    "description": "Usually used for storing equipment or items.",
+    "description": "Used for storing equipment or items.",
```

Comment:
I think this is meant to reference being able to hide in lockers, but I don't think it was added correctly. Check pillow fort for the `HIDE_PLACE` flag, but that would need to be a separate (but tiny) PR.

Reply:
> 
> 
> I think this is meant to reference being able to hide in lockers, but I don't think it was added correctly. Check pillow fort for the `HIDE_PLACE` flag, but that would need to be a separate (but tiny) PR.

Got it. I'll revert that.